server/src/main/java/org/elasticsearch/cluster/health/ClusterIndexHealth.java

```diff
@@ -100,7 +100,7 @@ public final class ClusterIndexHealth implements Writeable, ToXContentFragment {
         PARSER.declareInt(constructorArg(), new ParseField(ACTIVE_PRIMARY_SHARDS));
         PARSER.declareString(constructorArg(), new ParseField(STATUS));
         // Can be absent if LEVEL == 'indices' or 'cluster'
-        PARSER.declareNamedObjects(optionalConstructorArg(), SHARD_PARSER, new ParseField(SHARDS));
+        PARSER.declareNamedObjects(optionalConstructorArg(), SHARD_PARSER, new ParseField(ClusterStatsLevel.SHARDS.getLevel()));
```

Comment:
Hmm it is true that it's the same literal string, but that's something of a coincidence. I don't think we should link the field names in the response to the query parameters in the request like this.